### DISCUSSION TITLE
flomoPlugin

### DISCUSSION THREAD

**Discussion by extrastu** [flomoapp/3rd-party-tools#22] [分享创造]
# flomo for chrome
> 我的第一个chrome 插件竟然给了[flomo](http://flomoapp.com/)

#### 说明
已经运行有半个月，决定发出来
[https://github.com/it5200/flomoPlugin/tree/master/flomo-chrome](https://github.com/it5200/flomoPlugin/tree/master/flomo-chrome)

**目前还未上架，只能以压缩包形式安装**

#### v1.0.6 (开启UI优化之路)
- 优化popup

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm6sw4xwlnj30cc0c8aao.jpg)

#### v1.0.5（处于完善）
- 优化扩展页面无logo

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm4h6t9zu0j30b3062a9y.jpg)

#### v1.0.4（更快捷Send Text to flomo）
- 选中文字出现flomo 图标，点击即可发送到flomo(少楠的意见)

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm3pldp5hgj306b01wmwz.jpg)

#### v1.0.3 (在浏览器上随时随地的记录)
- 增加搜索建议，输入memo ，然后回车输入任意文字，点击enter即可保存到flomo
- 通知优化，使用chrome通知

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm3po6fpkzj312j06gjrj.jpg)

#### v1.0.2
- Save LInk to flomo 新增 title（预览如下图）

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm3ax2zhgyj30il02a744.jpg)

#### v1.0.1
- 初始版本提交

### 预览

![](https://tva1.sinaimg.cn/large/0081Kckwgy1glqy247st8j309p0am42a.jpg)

![](https://tva1.sinaimg.cn/large/0081Kckwgy1glqy34a31kj30rs0g3gyq.jpg)

![](https://tva1.sinaimg.cn/large/0081Kckwgy1glqy8vamj6j30ir05k74v.jpg)

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm3ax2zhgyj30il02a744.jpg)

### 使用说明

1、 下载/clone [https://github.com/it5200/flomoPlugin.git](https://github.com/it5200/flomoPlugin.git)

2、解压把 flomo-chome 拖入chrome 扩展程序 -> chrome://extensions/

3、右键flomo-chrome icon 点击选项输入flomo API 地址  

4、 配置好API地址，在任意网页右键即可看到Save Link to flomo / Save Text to flomo

### 更多
- 支持两种send to flomo（增加 **来自：**）
    - Save Link to flomo（当前网页地址 ）
    - Save Text to flomo（复制的文字 + 当前网页地址 ）

- 默认 #chrome 标签

PS：如果喜欢可以使用[我的链接🔗](https://flomoapp.com/register2/?NjM0)注册

### 最后

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm4rwcsj2uj30u00u0wfb.jpg)

![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm4ryijqvvj30u01fxe81.jpg)

**Comment by plidezus** [comment]
这个贼好用！

**Reply by konce** [reply]
很强
